pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -339,6 +339,8 @@ stdenv.mkDerivation {
       hardening_unsupported_flags+=" pic"
     '' + optionalString targetPlatform.isMinGW ''
       hardening_unsupported_flags+=" stackprotector"
+    '' + optionalString targetPlatform.isWindows ''
```

Comment:
We have so many of these we should provably move these to some attrset (not necessary now)